src/Viz.js

```diff
@@ -42,16 +42,20 @@ class Viz extends Component {
 
   /**
       @memberof Viz
-      @desc Sets visualization config, accounting for dataFormat, and renders the visualization.
+      @desc Sets visualization config, accounting for dataFormat, linksFormat, nodesFormat or topojsonFormat, and renders the visualization.
       @private
   */
   renderViz() {
     const {viz} = this;
-    const {config, dataFormat} = this.props;
+    const {config, dataFormat, linksFormat, nodesFormat, topojsonFormat} = this.props;
     const globalConfig = this.context.d3plus || {};
     const c = assign({}, globalConfig, config);
 
     if (dataFormat && c.data) viz.config(c).data(c.data, dataFormat);
+    if (linksFormat && c.links) viz.config(c).links(c.data, linksFormat);
+    if (nodesFormat && c.nodes) viz.config(c).nodes(c.data, nodesFormat);
+    if (topojsonFormat && c.nodes) viz.config(c).topojson(c.data, topojsonFormat);
```

Comment:
There are three problems with this new logic:
1. each new `if` statement you've added needs to pass the correct values to each method (so `.links(c.links` and not `.links(c.data`)
2. the last conditional should be checking for `c.topojson`, not `c.nodes` 
3. this logic might break if the user has a formatter function for 2 different methods, as each conditional always sets `.config(c)` (which might overwrite any previous formatter settings). I think the best thing would be to set `.config(c)` before any of these if statements, and then just let these if statements handle setting the specific methods (`data`, `nodes`, etc).